File: EPcar/src/ssr_pkg/src/messageSub_node.cpp
#include<ros/ros.h>
#include <std_msgs/String.h>

void message_callback(std_msgs::String msg)
{
    ROS_INFO(msg.data.c_str());
}

int main(int argc, char *argv[])
{
    ros::init(argc, argv, "messageSub_node");

    ros::NodeHandle nh;
    ros::Subscriber sub = nh.subscribe("qun",10, message_callback);

    while(ros::ok())
    {
        ros::spinOnce();
    }
    return 0;
}

File: EPcar/src/ssr_pkg/scripts/messageSub_node.py
#!/usr/bin/env python3

import rospy
from std_msgs.msg import String

def message_callback(msg):
    rospy.loginfo(msg)

if __name__ == "__main__":
    rospy.init_node("messageSub_node")
    rospy.logwarn("SUB!!")

    sub = rospy.Subscriber("qun",String,message_callback,queue_size=10)
    
    rospy.spin()
        
    exit(0)